Resources/Textures/Objects/Misc/flare.rsi/meta.json

```diff
@@ -0,0 +1,126 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "",
+    "copyright": "",
```

Comment:
These are missing a license and copyright. If you made them add CC-BY-SA-3.0 and your username.